src/high-level.js

```diff
@@ -13,6 +14,7 @@ export default function (options, javascriptScaffolderFactory) {
         languages: {
           [javaScriptLanguageChoice]: javascriptScaffolderFactory({
             [jsQuestionNames.PROJECT_TYPE]: 'Package',
+            [jsQuestionNames.PROJECT_TYPE_CHOICE]: PLUGIN_NAME,
```

Comment:
I know this wasn't part of your commit, but difference between `PROJECT_TYPE` and `PROJECT_TYPE_CHOICE` isn't clear to me given they're both being passed in as scaffolder choices. Does that somehow get passed down as a decision to the `Package` scaffolder?

Reply:
> difference between PROJECT_TYPE and PROJECT_TYPE_CHOICE isn't clear to me

yeah, naming is hard, haha. especially at the time these were named and i hadnt gotten even as consistent as i have at this point (which still isnt perfect).

> given they're both being passed in as scaffolder choices. Does that somehow get passed down as a decision to the Package scaffolder?

this important bit here is that they are being passed to the `javascriptScaffolderFactory`, not the js-scaffolder directly. 

the expected signature for such a factory is currently to take only the decisions map, like in https://github.com/pragmatic-divops/cli/blob/08273ab7bb39aff22e6bededec94e0d36ce8b102/src/common/enhanced-scaffolders.js#L7-L27

Reply:
another detail that isn't visible from this diff, but might be more related to what you could be asking about is that there are both `projectQuestionNames.PROJECT_TYPE` and `jsQuestionNames.PROJECT_TYPE` mentioned in this call. again, naming is hard, but the `projectQuestionNames.PROJECT_TYPE` existed long before i needed to consider app/package/cli distinction.

`projectQuestionNames.PROJECT_TYPE` really should be more along the lines of `projectQuestionNames.PROJECT_LANGUAGE`. the value of that constant [has been improved](https://github.com/travi/project-scaffolder/blob/29f782a187efe92d1878f418a1ce079c8c053c7d/src/prompts/question-names.js#L12_ but changing the constant name would be breaking, so i havent yet. i'm totally open to doing that sometime in a major version if you want to help me remember some time. might be a number of consumers to track down that are using it by now tho

Reply:
> the expected signature for such a factory is currently to take only the decisions map
that clarifies a bunch of things 👍 

I don't hate `PROJECT_TYPE` at all, but since you pointed it out 😄, I can't think of any projects that I would want to build that: 1) wouldn't be a language and 2) would need scaffolding like this. _sits another 5 minutes_ 🤔 now I'm thinking of something like a README project. I guess I could see some element of value in being able to create that project with a cli. Not sure it's enough value-add to have potential confusion every time someone sees `PROJECT_TYPE` though. And honestly if i was doing a readme project (say, an awesome list) I would probably want to use some sort of markdown linter and thus make it a JS project.

Reply:
> And honestly if i was doing a readme project (say, an awesome list) I would probably want to use some sort of markdown linter and thus make it a JS project.

as i was reading through this, this was going to be my response. this has been what my choice has been for that type of project so far (although i think there is an opportunity to call it out as a separate type within the js-scaffolder so that a few things could be better targeted. `docs-only` or similar is what has come to mind for me there in the past)